docs/config.md

```diff
@@ -868,6 +870,44 @@ The key through which access to the pinning service is granted
 
 Type: `string`
 
+#### `Pinning.RemoteServices: Policies`
+
+Contains additional opt-in policies for the remote pinning service.
+
+##### `Pinning.RemoteServices: Policies.MFS`
+
+When this policy is enabled, it follows changes to MFS
+and updates the pin for MFS root on the configured remote service.
+
+A pin request to the remote service is sent only when MFS root has changed
+and time has passed, since the previous pin is bigger than `RepinInterval`.
```

Comment:
@jessicaschilling  hm.. i think your question was valid, and I did not fully get what you meant when I merged this.

IPFS node  checks if MFS changed every 30s and if it did change, then it checks when the last successful pin was executed.

If `last_pin_time` + `RepinInterval` > `now`, then it waits until it becomes otherwise.
In other words, we never send pin request more often than the value in `RepinInterval` (which is 5 minutes by default).

Would below be better?

```suggestion
A pin request to the remote service is sent only when MFS root CID has changed
and enough time has passed since the previus request (determined by `RepinInterval`).
```

Reply:
Makes perfect sense, thank you!
one note: `previous` 😊